extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -141,10 +207,313 @@ def get(self, objectId: str = None):
             _err = traceback.format_exc()
             return self.error(f'failure: {_err}')
 
+    @auth_or_token
+    async def post(self, objectId):
```

Comment:
why async?

Reply:
because I am calling K asynchronously and awaiting the results; you can `await` only inside an `async` function.